### PR TITLE
Integration of Astar Parachain and Update of Zombienet Configuration

### PR DESCRIPTION
**Pull request by dastanfv**
## PR Type:
Enhancement

___
## PR Description:
This PR integrates the Astar parachain into the project by updating the zombienet configuration and the build process. It also increases the timeout for the zombienet connection and updates the README file with new instructions.

___
## PR Main Files Walkthrough:
<details> <summary>files:</summary>

`.github/workflows/build.yml`: The build process is updated to download and set up the Astar parachain binary. The environment variables for zombienet are also updated to include the Astar parachain command. The timeout for the zombienet connection is increased.
`ownership-chain/zombienet/README.md`: The README is updated with instructions on how to set up and run the Astar parachain along with the Ownership Chain. It also explains that the setup comes with an established hrmp channel between the two parachains.
`ownership-chain/zombienet/native.toml`: The zombienet configuration file is updated to include the Astar parachain. The id for the Ownership Chain is changed and a new parachain entry for Astar is added. Two hrmp channels are also added for communication between the two parachains.
</details>

### CODE REVIEW

File: ownership-chain/zombienet/native.toml [hunk 1]
```diff
@@ -12,12 +12,35 @@ chain = "rococo-local"
...
+  [[parachains.collators]]
+  name = "Alice"
+  ws_port = 9998
+  command = "{{ZOMBIENET_ASTAR_COMMAND}}"
+  validator = true
+
+[[hrmp_channels]]
+sender = 2900
+recipient = 2008
+max_capacity = 8
```
Comment (dastanfv) [inline ownership-chain/zombienet/native.toml:39]:
```bash
`max_capacity` - specifies how many messages can be in the channel at once.
`max_message_size` - specifies the maximum size of the messages.
```

File: ownership-chain/zombienet/native.toml [hunk 2]
```diff
@@ -12,12 +12,35 @@ chain = "rococo-local"
...
+  command = "{{ZOMBIENET_OWNCHAIN_COMMAND}}"
   validator = true
+
+[[parachains]]
+id = 2008
+cumulus_based = true
+chain = "astar-dev"
+
+  [[parachains.collators]]
+  name = "Alice"
```
Comment (asiniscalchi) [inline ownership-chain/zombienet/native.toml:31]: let's use simply "astar"

File: ownership-chain/zombienet/native.toml [hunk 3]
```diff
@@ -12,12 +12,35 @@ chain = "rococo-local"
   validator = true
 
 [[parachains]]
-id = 1000
+id = 2900
 cumulus_based = true
 chain = "local-v"
 
   [[parachains.collators]]
   name = "Alice"
```
Comment (asiniscalchi) [inline ownership-chain/zombienet/native.toml:20]: let's use "ownership" or something better than Alice if possible:
image

**Review by asiniscalchi — CHANGES_REQUESTED** [review]

**Review by asiniscalchi — APPROVED** [review]

**Pull request merged**